src/stack.js

```diff
@@ -1,5 +1,49 @@
-'use strict'
-
 export default class Stack {
-  // your code here
+  constructor() {
+    this.commissary = [],
+    this.size = 0,
+    this.topIndex = null
```

Comment:
It looks like `size` is always `topIndex + 1` except when `size === 0` and `topIndex === null`.  Maybe you could use only one variable and get rid of some logic.  (You never test for `topIndex === null`)